src/ControlSystem/Tags.hpp

```diff
@@ -10,23 +10,124 @@
 #include <unordered_map>
 #include <utility>
 
+#include "ControlSystem/Averager.hpp"
+#include "ControlSystem/Controller.hpp"
+#include "ControlSystem/TimescaleTuner.hpp"
 #include "DataStructures/DataBox/Tag.hpp"
 #include "DataStructures/DataVector.hpp"
 #include "Domain/Creators/DomainCreator.hpp"
 #include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
 #include "Domain/FunctionsOfTime/PiecewisePolynomial.hpp"
 #include "Domain/OptionTags.hpp"
+#include "Options/Options.hpp"
 #include "Time/Tags.hpp"
 #include "Utilities/ErrorHandling/Error.hpp"
 #include "Utilities/MakeWithValue.hpp"
 #include "Utilities/TMPL.hpp"
 
-namespace control_system::Tags {
-/// The measurement timescales associated with
-/// domain::Tags::FunctionsOfTime.  Each function of time associated
-/// with a control system has a corresponding set of timescales here,
-/// represented as `PiecewisePolynomial<0>` with the same components
-/// as the function itself.
+namespace control_system {
+namespace OptionTags {
+/// \ingroup OptionTagsGroup
+/// \ingroup ControlSystemGroup
+/// Options group for all control system options
+struct ControlSystemGroup {
+  static std::string name() noexcept { return "ControlSystem"; }
+  static constexpr Options::String help = {"Control system"};
+};
+
+/// \ingroup OptionTagsGroup
+/// \ingroup ControlSystemGroup
+/// Options for the averager
+template <size_t DerivOrder>
+struct Averager {
+  using type = ::Averager<DerivOrder>;
+  static constexpr Options::String help = {"Averager"};
+  using group = OptionTags::ControlSystemGroup;
+};
+
+/// \ingroup OptionTagsGroup
+/// \ingroup ControlSystemGroup
+/// Options for the controller
+template <size_t DerivOrder>
+struct Controller {
+  using type = ::Controller<DerivOrder>;
+  static constexpr Options::String help = {"Controller"};
```

Comment:
These structs should have better help text, even if just the same bland sentence as the doxygen description of the struct.